tutorials/pyroot/pyroot005_contextmanagers.py

```diff
@@ -0,0 +1,104 @@
+# \file
+# \ingroup tutorial_pyroot
+# \notebook -nodraw
+#
+# This tutorial demonstrates the usage of the TContext and TFile classes as
+# Python context managers.
+#
+# \macro_code
+# \macro_output
+#
+# \date March 2022
+# \author Vincenzo Eduardo Padulano CERN/UPV
+import os
+
+import ROOT
+from ROOT import TDirectory, TFile
+
+# By default, ROOT objects like histograms, graphs etc. are automatically attached
```

Comment:
Ok sorry about the graph misconception! I think I can just stick to examples with histograms for this tutorial to avoid making it too complex. Maybe showing also other C++ classes that are not owned by the file could be an idea.